src/DynamoPackages/PackageManagerClient.cs

```diff
@@ -276,6 +289,45 @@ internal void Publish(Package package, IEnumerable<string> files, IEnumerable<st
             }
         }
 
+        internal void PublishRetainFolderStructure(Package package, IEnumerable<IEnumerable<string>> files, IEnumerable<string> markdownFiles, bool isNewVersion, PackageUploadHandle packageUploadHandle)
```

Comment:
Add comment to this internal method. Would be useful in future.

Reply:
Yup, added a detailed comment for this method.